src/common.js

```diff
@@ -111,6 +115,7 @@ exports.ShellString = ShellString;
 
 // Return the home directory in a platform-agnostic way, with consideration for
 // older versions of node
+// XXX: I don't think this should be a part of plugin.utils. Not sure what should happen to it. Maybe just inline it.
```

Comment:
Hmm... Would you be ok creating a `polyfills.js` file which contains all of our polyfills?